oncall/store.go

```diff
@@ -196,7 +197,7 @@ func (s *Store) OnCallUsersBySchedule(ctx context.Context, scheduleID string) ([
 }
 
 // HistoryBySchedule will return the list of shifts that overlap the start and end time for the given schedule.
-func (s *Store) HistoryBySchedule(ctx context.Context, scheduleID string, start, end time.Time) ([]Shift, error) {
+func (s *Store) HistoryBySchedule(ctx context.Context, scheduleID string, start, end time.Time, userIDs []string) ([]Shift, error) {
```

Comment:
I'm thinking since this is already a complicated method, we move the user ID filtering to the caller (graphql code).

It's a relatively simple thing to do and then we can avoid having to update methods, tests, etc...

Filtering is done at the very end, and is how we're already doing it elsewhere like calendar subscriptions.